packages/api/src/controllers/animalBatchController.js

```diff
@@ -116,48 +117,28 @@ const animalBatchController = {
           // TODO: allow animal group editing
           // await checkAndAddGroup(req, animal, farm_id, trx);
 
-          const {
-            id,
-            count,
-            custom_breed_id,
-            custom_type_id,
-            default_breed_id,
-            default_type_id,
-            name,
-            notes,
-            photo_url,
-            organic_status,
-            supplier,
-            price,
-            sex_detail,
-            origin_id,
-            group_ids,
-            animal_batch_use_relationships,
-          } = animalBatch;
-
-          await baseController.upsertGraph(
-            AnimalBatchModel,
-            {
-              id,
-              count,
-              custom_breed_id,
-              custom_type_id,
-              default_breed_id,
-              default_type_id,
-              name,
-              notes,
-              photo_url,
-              organic_status,
```

Comment:
This was causing a not-null constraint on organic_status despite being undefined with very plain patches { id, default_type_id } . Commenting out this line fixes it but I have no clue why... real mystery here for me.